### ISSUE TITLE
CMake build issue for debug build on Windows

### ISSUE THREAD

**Issue by kant2002** [ihhub/fheroes2#3216]
Hi @winterheart can you support me in troubleshooting the issues with CMake build
I follow instruction and build FHeroes2, but unfortunately I receive error  
```
Не удается продолжить выполнение кода, поскольку система не обнаружила SDL2.dll. Для устранения этой проблемы попробуйте переустановить программу. 
```
I compile application in Debug mode, but in the output folder sitting only `SDL2d.dll`. Do you know what's going on?

**Comment by winterheart** [comment]
Hello. There some issues on building SDL2 libraries in vcpkg in Debug mode. Some libraries are linked with Release version of SDL2 which has SDL2.dll name. As temporary workaround just copy SDL2d.dll as SDL2.dll, and this will work.

**Comment by ihhub** [comment]
Possibly fixed in #7620
